Sources/SwiftParser/Statements.swift

```diff
@@ -208,7 +208,7 @@ extension Parser {
       // However, if this is the first clause, and we see "x = y", then this is
       // almost certainly a typo for '==' and definitely not a continuation of
       // another clause, so parse it as an expression.  This also avoids
-      // lookahead + backtracking on simple if conditions that are obviously
+      // lookahead + lookaheading on simple if conditions that are obviously
       // boolean conditions.
```

Comment:
```suggestion
      // lookahead on simple if conditions that are obviously boolean conditions.
```

Reply:
Thank you for your suggestion regarding the comments on lines 211 to 212 in Statements.swift. I've made the requested modification as per your recommendation.